docs/playground/src/PlaygroundConfiguration.ts

```diff
@@ -79,7 +79,9 @@ export const PlaygroundConfiguration: IStringMap<IComponentPlaygroundConfigurati
   ErrorReport: {
     show: true,
     toExecute: () => {
-      Coveo['SearchEndpoint'].endpoints['default'].options.accessToken = 'invalid';
+      $$(getSearchInterfaceElement()).on('buildingQuery', function(e, args) {
+        args.queryBuilder.pipeline = 'invalid';
```

Comment:
Hmm, I'm curious why we need to set the pipeline to `invalid`?

Reply:
Ok, I saw in the main thread that it was to force an error, but wouldn't it be safer to use the `accessToken` approach? Someone might one day create a pipeline called 'invalid' for example.

Reply:
Done. I've overwritten the search endpoint.